utils/config/daos.yml

```diff
@@ -1,42 +1,45 @@
-# DAOS agent configuration file.
+# DAOS client configuration file.
 #
 # Location of this configuration file is determined by first checking for the
-# path specified through the -f option of the daos_agent command line.
-# Otherwise, /etc/daos.conf is used.
+# path specified through the -o option of the daos_agent and DMG command line.
+# Otherwise, etc/daos.yml is used.
 #
 # Section describing the client configuration
 #
 # Although not supported for now, one might want to connect to multiple
 # DAOS installations from the same node in the future.
 #
 # Specify the associated DAOS systems.
-# Name must match name specifed in the daos_server.conf file on the server.
-#
-systems:
--
-  name: daos
-#
-#  # Access points
-#  # To operate, DAOS will need a quorum of access point nodes to be available.
-#  # Hosts can be specified with or without port, default port below.
-#  #
-#  # access_pts: ['hostname1:10001','hostname2:10001','hostname3:10001']
-#  # access_pts: [hostname1,hostname2,hostname3]
-#
-#  # Force default port number to connect to on daos_server, this will also
-#  # be used when connecting to access points if no port is specified.
-#  #
-#  # default: 10000
-#  # port: 10001
-#
-#  # Path to CA certificate
-#  # Not required if the DAOS server is configured in an insecured way
-#  #
-#  # ca_cert: ./.daos/ca.crt
-#
-#  # Path to client certificate and key file.
-#  # Discarded if no CA certificate is passed.
-#  #
-#  # default: ./.daos/daos.{crt,key}
-#  # cert: ./.daos/daos.crt
-#  # key: ./.daos/daos.key
+# Name must match name specifed in the daos_server.yml file on the server.
+#
+# default: daos_server
```

Comment:
The default name in daos_server.yml is "daos" - we should be consistent with the defaults between the two. I'd lean toward changing the code and this file to use "daos" as a default, since this is the DAOS system (cluster) name not some individual server. @johannlombardi should be the decider on this, though.

Reply:
The daos_server.yml uses a commented out "name" of "daos", but the built in default value in config_types.go specifies SystemName: "daos_server".  I've changed the built in default value for the client to match "daos_server" and left the commented out value "daos" as we did in the server's yml.   Am I conflating two different fields?

Reply:
I think probably @kjacque is correct but should change all occurrences in one patch and leave this consistent with the server at the time of landing. @jbrosenz can you please create a ticket for this, thanks

Reply:
I've created a new story here:  https://jira.hpdd.intel.com/browse/DAOS-2395